tests/components/kostal_plenticore/test_number.py

```diff
@@ -90,108 +89,3 @@ async def test_setup_no_entries(
     ent_reg = async_get(hass)
     assert ent_reg.async_get("number.scb_battery_min_soc") is None
     assert ent_reg.async_get("number.scb_battery_min_home_consumption") is None
-
-
```

Comment:
@MartinHjelmare suggested in #64927 not to interact with integration details. Unfortunately I found no way to keep this test when setting up the integration. The main problem was not to setup the integration and platform but to update the state of the entity.

Any suggestions are welcome.

Reply:
Patch the library and then move time forward to have the coordinator fetch new data from the patched library.

https://github.com/home-assistant/core/blob/d0c4d39ae20841b441b497cdd42e6b48f6162c2c/tests/common.py#L375-L378